lib/database/user.php

```diff
@@ -617,14 +597,16 @@ function getUserForumPostAuth($user)
 
     if ($dbResult !== false) {
         $data = mysqli_fetch_assoc($dbResult);
-        return $data['ManuallyVerified'];
+
+        return (bool) $data['ManuallyVerified'];
```

Comment:
This casts `null`/`""`/`0`/`"0"` to `false` and `1`/`"1"` to `true`. 
Nullable bool would result in the same behaviour but since this function's logic allows for a clean `bool` return type i went for the cast.

Reply:
I was concerned about the "null" case, but while the field is nullable in the DB, I don't have any null entries.